### PR TITLE
fix: Update ObjectStatus colors

### PR DESCRIPTION
**Pull request by salarenko**
## Related Issue
Closes: #956

## Description
This PR updates inverted Object status colors.

#### Please check whether the PR fulfills the following requirements

- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-styles/wiki/PR-Review-Checklist

### CODE REVIEW

File: src/functions/_deep-get.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/// Map deep get
+/// @author Hugo Giraudel
+/// @access public
+/// @param {Map} $map - Map
```
[the remark this answers is not in the document]
Comment (jbadan) [reply src/functions/_deep-get.scss:4]: I don't think we can have this - if we want to use someone else's work it needs to have an open source license.

[the remark this answers is not in the document]
Comment (mikerodonnell89) [reply src/functions/_deep-get.scss:4]: https://css-tricks.com/license/

I wouldn't worry about it

[the remark this answers is not in the document]
Comment (jbadan) [reply src/functions/_deep-get.scss:4]: As cute as that is, I don't think it would pass a SAP scan 😆

[the remark this answers is not in the document]
Comment (salarenko) [reply src/functions/_deep-get.scss]: I removed author annotations. This util is quite common use-case

**Comment by salarenko** [comment]
> Removing the author's name isn't a solution

What is the solution then? We cannot use some common way of resolving a problem because someone did it first and published it on the internet? I'd like to mention the author but you are saying it wont pass "SAP scan" and I cannot use it without authoring, event tho this is a for loop with return.

**Comment by jbadan** [comment]
@salarenko I'm not sure what the solution should be. Have you checked for an open sourced licensed version of this function so we can avoid this enitrely? I could be wrong here, but this is a grey area and feels wrong to include with just the author's name removed. How about looping in the open source office and getting some clarity on what to do in this situation? I would try #sap-open-source in slack

**Comment by droshev** [comment]
@salarenko let's take this `offline` :)

**Comment by droshev** [comment]
We agreed to refactor the code and not to have the copy

**Pull request merged**